### PR TITLE
Add Chapter 08 notes

### PR DESCRIPTION
**Pull request by ivelasq**
Hello! Adding my notes (sorry it's a bit late).

### CODE REVIEW

File: 08-package-metadata.Rmd [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+## Meeting Videos
+
+### Cohort 1
+
+`r knitr::include_url("https://www.youtube.com/embed/URL")`
+
+<details>
+<summary> Meeting chat log </summary>
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/vQSYeGwX9yQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (jonthegeek) [inline 08-package-metadata.Rmd]: It looks (from this plus from some of the code blocks) like you copied a rendered md into this Rmd. Or maybe you were trying to sort out how the video gets embedded? You need to put the `vQSYeGwX9yQ` part into `URL` in the `knitr::include_url` piece above.

Everything looks fine (as long as it renders) other than this bit, but it doesn't look like standard Rmarkdown code chunks.

**Review by jonthegeek — CHANGES_REQUESTED** [review]
See comment about the youtube embed.

File: 08-package-metadata.Rmd (hunk 1 above)
Comment (ivelasq) [reply 08-package-metadata.Rmd]: Whoops, apologies! I think I just have Quarto on the brain.

**Review by jonthegeek — APPROVED** [review]
Looks good, assuming the build works!

**Pull request merged**